src/Core/src/Core/Extensions/VisualTreeElementExtensions.cs

```diff
@@ -52,70 +53,64 @@ static IList<IVisualTreeElement> GetVisualTreeDescendantsInternal(this IVisualTr
 		}
 
 		/// <summary>
-		/// Gets list of a Visual Tree Elements children based off of a given x, y point.
+		/// Gets list of a Visual Tree Elements children based off of a rectangle defined by it's coordinates.
```

Comment:
nit: "its coordinates"

Reply:
Please update <summary/> comment to specify that coordinates are in platform units (as opposite to pixels). For all methods which take positional coordinates.

Reply:
We need such comment for all public methods